src/eth/instance/kubernetesCluster.ts

```diff
@@ -20,7 +20,18 @@ export default class KubernetesInstance extends AbstractInstance {
   public async delete (payload: ClusterDeleteType) {
     logger.debug('Kubernetes instance delete')
     if (this.kubernetesInfra !== undefined) {
-      return await this.kubernetesInfra.deleteDeploymentAndService(payload)
+      try {
+        await this.kubernetesInfra.deleteNamespace(payload.namespace)
+        logger.info(`Successfully deleted namespace ${payload.namespace}`)
+      } catch (error) {
+        logger.error(`Failed to delete namespace ${payload.namespace}:`, error)
+        try {
+          await this.kubernetesInfra.forceDeleteNamespace(payload.namespace)
+          logger.info(`Successfully force deleted namespace ${payload.namespace}`)
+        } catch (forceError) {
+          logger.error(`Failed to force delete namespace ${payload.namespace}:`, forceError)
+        }
+      }
```

Comment:
try catch in try catch not good

Reply:
in here try catch is so weird, you should put in 
`deleteNamespace` or `forceDeleteNamespace`